pkg/exchange/coinbase/stream.go

```diff
@@ -47,8 +47,9 @@ type Stream struct {
 	orderbookSnapshotMessageCallbacks []func(m *OrderBookSnapshotMessage)
 	orderbookUpdateMessageCallbacks   []func(m *OrderBookUpdateMessage)
 
-	authEnabled   bool
-	userOrderOnly bool
+	bbgoChannelsOnly    bool // if true, only subscribe to bbgo channels on connect. Otherwise, the user can subscribe to both bbgo and coinbase channels
```

Comment:
for doc comment, we use this convention, please take a look:
https://tip.golang.org/doc/comment